app/views/school_groups/chart_updates/index.html.erb

```diff
@@ -1,3 +1,11 @@
+<% content_for :page_title, @school_group.name + ' | ' +  t('school_groups.chart_updates.index.group_chart_settings').capitalize %>
+<%= render 'school_groups/sub_nav' if can?(:update_settings, @school_group) %>
```

Comment:
Is there a more consistent way we can ensure this appears on all school group pages? 

Currently, we've got an explicit check for the school groups controller in `application.html.erb` (`controller_name == 'school_groups' && can?(:update_settings, @school_group)`) and then we have this check here in a specific template. 

Am wondering if we can add a more generic test / helper in the main template, e.g. to check if this is a controller in the SchoolGroups module, and they have update settings?

Reply:
:+1: have moved the check to display the group pages sub nav to application.html.erb